src/tests/ftest/lustre/llmount.py

```diff
@@ -0,0 +1,337 @@
+        """
+
+        self.config_write()
+
+    # return 0 if directory successfully created
+    def create_unique_directory(self, directory):
+        self.dirtest = self.mountpoint + "/" + str(directory)
+        launch_cmd = " mkdir " + self.dirtest
+        clientsArr = self.clients.split(",")
+
+        rc = subprocess.call(shlex.split("ssh " + clientsArr[0] + launch_cmd))
+        return rc
+
+    def results(self,ldir):
+        encoding = "UTF-8"
+        skipped = passed = fails = 0
+
+        cmd = "grep -c 'name: test' " + self.lustretestslogdir + ldir + "/results.yml"
+        nbtest = subprocess.check_output(shlex.split(cmd))
+        nbtests = int(nbtest.rstrip().decode(encoding))
+
+        cmd = "grep -c SKIP " + self.lustretestslogdir + ldir + "/results.yml"
+        try:
+            skipp = subprocess.check_output(shlex.split(cmd))
+        except Exception,e:
+            skipp = str(e.output)
+        skipped = int(skipp.rstrip().decode(encoding))
+
+        cmd = "grep -c PASS " + self.lustretestslogdir + ldir + "/results.yml"
+        try:
+            passp = subprocess.check_output(shlex.split(cmd))
+        except Exception,e:
+            passp = str(e.output)
+        passed = int(passp.rstrip().decode(encoding))
+
+        cmd = "grep -c FAIL " + self.lustretestslogdir + ldir + "/results.yml"
+        try:
+            failedp = subprocess.check_output(shlex.split(cmd))
+        except Exception,e:
+            failedp = str(e.output)
+        fails = int(failedp.rstrip().decode(encoding))
+
+        print("SKIP: " + str(skipped) + "\tPASS: " + str(passed) + "\tFAILED: " + str(fails) +
+              "\tTotal Tests: " + str(nbtests))
+        if fails is not 0:
+            self.fail("Some sanity tests failed: " + fails + " / " + nbtests)
+
+    # run test suites with auster
+    # environment variables must be saved then modified with os.environ.copy
+    # passing env = {} in subprocess.call/popen will break auster PATH for some reason
+    def test_lustre_auster(self):
+        """
+        run lustre specified tests suites
+
+        :avocado: tags=lustre,auster
+        """
+        DIR_UUID = uuid.uuid4()
+        if self.create_unique_directory(DIR_UUID) is not 0:
+            self.fail("Failed to create test directory " + str(DIR_UUID) + " in " + self.mountpoint)
+
+        austerparams = " -f " + self.lfsname + " -v "
+        if self.slow:
+            austerparams += " -s "
+        austerparams += " -D " + self.lustretestslogdir + str(DIR_UUID)
+        for suite in self.test_suites:
+            austerparams += " " + str(suite.name) + " --only \"" + str(suite.only) + "\""
+            austerparams += " --except \"" + str(suite.excepts) + "\""
+
+        print self.ltestpath + "auster " + austerparams
+        envcpy = os.environ.copy()
+        envcpy["DIR"] = self.dirtest
+        subprocess.call([self.ltestpath + "auster"] + shlex.split(austerparams), env=envcpy)
+        self.results(str(DIR_UUID))
```

Comment:
So not really testing anything, do you not need to look at results of the call at all?